sys/fuchsia/clock.txt

```diff
@@ -4,13 +4,17 @@
 include <zircon/syscalls.h>
 include <zircon/syscalls/clock.h>
 
+resource zx_clock[zx_handle]
 resource zx_time[int64]: 0, ZX_TIME_INFINITE
 
+# ZX_CLOCK_ARGS_VERSIONV1 0x400000000000000
```

Comment:
Nit: Can you add a comment with a little bit of explanation about why these are here as comments, and not consts?

Reply:
FWIW I initially tried that way, and caused some bugs, but now it doesn't 🤔 
so now I'm adding them back as constants 🤷